test/controllers/DeparturesControllerSpec.scala

```diff
@@ -493,6 +505,62 @@ class DeparturesControllerSpec extends SpecBase with ScalaCheckPropertyChecks wi
         val request = FakeRequest(GET, routes.DeparturesController.get(DepartureId(1)).url)
         val result  = route(application, request).value
 
+        status(result) mustEqual INTERNAL_SERVER_ERROR
+      }
+    }
+  }
+  "/GET" - {
```

Comment:
Could we just have method name here (getDepartures)

Reply:
That would be possible but then we would break the naming pattern in that spec comparing to other test cases.

Reply:
Thanks @marcinkrykowski, i didn't realise we were using a slightly different naming convention for the controller specs 👍